solver/src/solver/single_order_solver.rs

```diff
@@ -104,40 +89,6 @@ impl<I: SingleOrderSolving + Send + Sync + 'static> Solver for SingleOrderSolver
     }
 }
 
-impl<I: SingleOrderSolving + Send + Sync> SingleOrderSolver<I> {
-    /// Return a settlement for the given limit order (if possible)
-    pub async fn settle_order(&self, order: LimitOrder) -> Result<Option<Settlement>> {
-        let solver_name = self.inner.name();
-        let max_retries = 2;
-        let mut errors = vec![];
-        for _ in 0..max_retries {
-            match self.inner.try_settle_order(order.clone()).await {
-                Ok(settlement) => return Ok(settlement),
-                Err(err) if err.retryable => {
-                    tracing::debug!("Retrying {} settlement due to: {:?}", solver_name, &err);
-                    errors.push(err.inner);
-                    continue;
-                }
-                Err(err) => return Err(err.inner),
-            }
-        }
-        // One last attempt, else throw converted error
-        self.inner.try_settle_order(order).await.map_err(|err| {
-            errors.push(err.inner);
-            anyhow!(format!(
-                "{} Errored after {} attempts. Accumulated errors follow \n{}",
-                solver_name,
-                max_retries + 1,
-                errors
-                    .iter()
-                    .enumerate()
-                    .map(|(i, err)| format!("Attempt {}: {}", i + 1, err.to_string()))
-                    .join("\n")
```

Comment:
What happened to our pretty accumulated error log?

Reply:
I removed it because it made sense when we were retrying the same order several times in a row but not with this logic. Retryable errors get warn logged and unretryable error logged.